ipfshttpclient/client/__init__.py

```diff
@@ -240,7 +240,13 @@ def apply_workarounds(self):
 	
 	@utils.return_field('Hash')
 	@base.returns_single_item(dict)
-	def add_bytes(self, data: bytes, **kwargs):
+	def add_bytes(self, data: bytes
+	        trickle: bool = False, only_hash: bool = False,
+	        chunker: ty.Optional[str] = None,
+	        pin: bool = True, raw_leaves: bool = None,
+	        cid_version: ty.Optional[int] = None,
+	        **kwargs: base.CommonArgs):
```

Comment:
If you tell me what to put I'll copy/paste it :)

Reply:
No worries. There's some examples in this library of how to do it. The library pre-dates Python's typing support so there's also lots of code that doesn't use it yet, or uses it a legacy way (e.g. with comments).

Function that doesn't return anything:

`def foo(bar: str) -> None:`

Function that returns a value:

`def foo(bar: str) -> int:`

Function that returns a tuple:
 
`def foo(bar: str) -> ty.Tuple[int, bytes]:`

Note that the `ty.` prefix is unique to this library since all the `typing` imports are imported as follows:

`import typing as ty`

Reply:
Done.